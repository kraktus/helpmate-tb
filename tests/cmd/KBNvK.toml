bin.name = "tb"
args = "-m \"KBNvK\" --no-time"