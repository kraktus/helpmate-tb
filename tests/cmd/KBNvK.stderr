dhat: Total:     118,445,653 bytes in 822 blocks
dhat: At t-gmax: 20,567,089 bytes in 66 blocks
dhat: At t-end:  1,571 bytes in 7 blocks
dhat: The data has been saved to dhat-heap.json, and is viewable with dhat/dh_view.html
